widget/embedded/src/store/slices/data.ts

```diff
@@ -42,14 +45,17 @@ export const createDataSlice: StateCreator<
   DataSlice
 > = (set, get) => ({
   // State
-  _blockchains: [],
+  _blockchainsMapByName: null,
```

Comment:
if you set `new Map()`, you can get read of `blockchainsMapByName?.values() || []` and use `blockchainsMapByName.values()`. Do you have any usecase for `null`?

Reply:
Fixed